plugins/lock.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Redis = require('ioredis');
+const Redlock = require('redlock');
+const config = require('config');
+
+class Lock {
+    constructor() {
```

Comment:
probably should add jsdoc like this: https://github.com/screwdriver-cd/executor-k8s/blob/master/index.js#L163-L218